licenses/licenses_tests.py

```diff
@@ -333,17 +352,17 @@ def test__licenses__views__FilledLicenseFile__3(
     assert 'License not found.' in browser.contents
 
 
-def test__licenses__views__Filled_licenseFile__4(
-        browser, license_template_dict):
-    """The user of a LR needs to have a profile."""
-    user = User.objects.create_user(testing.EMAIL, password=testing.PWD)
-    lr = create_license_request(
-        user, default_category(), license_template_dict)
+# def test__licenses__views__FilledLicenseFile__4(
+#         browser, license_template_dict):
+#     """The user of a LR needs to have a profile."""
+#     user = User.objects.create_user(testing.EMAIL, password=testing.PWD)
+#     lr = create_license_request(
+#         user, default_category(), license_template_dict)
 
-    browser.login()
-    browser.open(print_url(lr.id))
+#     browser.login()
+#     browser.open(print_url(lr.id))
 
-    assert "There is no profile" in browser.contents
+#     assert "There is no profile" in browser.contents
```

Comment:
Commented-out code in the repository should be the exception, and if so, an explanatory comment should be provided explaining why the code is commented out.